backend/hitas/services/thirty_year_regulation.py

```diff
@@ -632,9 +632,31 @@ def build_thirty_year_regulation_report_excel(results: ThirtyYearRegulationResul
         )
         worksheet.append(data)
 
+    last_row = worksheet.max_row
+    worksheet.auto_filter.ref = worksheet.dimensions
+
+    worksheet.append(
+        ReportColumns(
+            display_name="",
+            acquisition_price=f"=SUM(B2:B{last_row})",
+            apartment_count=f"=SUM(C2:C{last_row})",
+            indices="",
+            change=f"=SUM(E2:E{last_row})",
+            adjusted_acquisition_price=f"=SUM(F2:F{last_row})",
+            surface_area=f"=SUM(G2:G{last_row})",
+            price_per_square_meter=f"=SUM(H2:H{last_row})",
```

Comment:
Should `price_per_square_meter` be also averaged, as the sum value is not really that useful I would imagine 🤔

Reply:
Yes, that would be a good idea. I'm thinking it might be useful to have a "result row" with separate titles, one row under the rest of the rows. That way we could make more explicit what each summary means.

Reply:
Added an extra row between the housing company data and the summary, so that excel filtering and sorting works properly.